apps/evm/src/clients/api/queries/getMerklRewards/formatMerklRewardsResponse.ts

```diff
@@ -0,0 +1,139 @@
+import BigNumber from 'bignumber.js';
+import type { MerklDistribution, Token } from 'types';
+import { areAddressesEqual } from 'utilities';
+import type { GetMerklUserRewardsResponse, MerklRewardBreakdown } from '.';
+import type { PendingExternalRewardSummary } from '../getPendingRewards/types';
+
+type MerklCampaign = MerklDistribution & { poolComptrollerAddress: string };
```

Comment:
`poolComptrollerAddress` isn't used, so I believe it can be removed (and you can use the `MerklDistribution` type directly then).